docs/extending/customising_group_views.md

```diff
@@ -106,3 +106,27 @@ INSTALLED_APPS = [
     ...,
 ]
 ```
+
+## Customising the Group Editor Object Permissions Order
```

Comment:
```suggestion
## Customising the group editor permissions ordering
```

Just simplifying and removing the article casing on the titles, we usually avoid this and just title case proper nouns.

Reply:
Changed in [1c515e4](https://github.com/wagtail/wagtail/pull/10469/commits/1c515e4d49262d2a8ed80eb201fa8046670563be)